tools/stimulus/functions-R.yaml

```diff
@@ -27,7 +27,7 @@ igraph_ecount:
     IGNORE: RR, RC
 
 igraph_neighbors:
-    IGNORE: RR, RC
+    DEPS: vid ON graph
```

Comment:
When you need to add a DEPS line to make things work, consider submitting an upstream PR to do it directly in the C core sources. (I'm going to do this one.)

Reply:
I hope I didn't mess anything up, here are some more DEPS added:

https://github.com/igraph/igraph/commit/bcb8fbcfa7dfaeed02a5f4472eae36513c716f29

These won't show up here until the auto-vendoring is resumed.